src/Benchmarks/Benchmarks.csproj

```diff
@@ -11,6 +11,8 @@
     <ApplicationManifest Condition="$(UseWinmd) == true">Benchmarks.manifest</ApplicationManifest>
     <BenchmarkTargetFramework>$(TargetFramework)</BenchmarkTargetFramework>
     <BenchmarkTargetFramework Condition="'$(BenchmarkTargetFramework)' == 'netcoreapp2.0'">netstandard2.0</BenchmarkTargetFramework>
+    <IsDotnetBuild Condition="'$(IsDotnetBuild)' == ''">false</IsDotnetBuild>
+	  <LangVersion Condition="$(IsDotnetBuild) == true">9.0</LangVersion>
```

Comment:
indenting is off.

Reply:
@ujjwalchadha, please change your editor setting to use spaces instead of tabs